src/libs/RequestThrottle.ts

```diff
@@ -22,11 +25,15 @@ function getLastRequestWaitTime() {
     return requestWaitTime;
 }
 
-function sleep(): Promise<void> {
+function sleep(error: RequestError, command: string): Promise<void> {
     requestRetryCount++;
     return new Promise((resolve, reject) => {
         if (requestRetryCount <= CONST.NETWORK.MAX_REQUEST_RETRIES) {
-            setTimeout(resolve, getRequestWaitTime());
+            const currentRequestWaitTime = getRequestWaitTime();
+            Log.info(
+                `[RequestThrottle] Retrying request after error: '${error.name}', '${error.message}'. Command: ${command}. Retry count:  ${requestRetryCount}. Wait time: ${currentRequestWaitTime}`,
```

Comment:
Is there a way to add the error code here? As well as the json code?

Reply:
I added the error code :)